src/Infocom.php

```diff
@@ -185,10 +185,12 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
      **/
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        if (!$item instanceof CommonDBTM) {
+            return false;
+        }
 
-        switch ($item->getType()) {
-            case 'Supplier':
-                /** @var Supplier $item */
+        switch (true) {
```

Comment:
This could just be an if statement.

Reply:
I've noticed and so has my IDE. In this case though, you replaced the switch condition with `true` so it really isn't any different than an if statement even if more conditions get added as cases.